__tests__/app.test.js

```diff
@@ -95,4 +95,46 @@ describe('/api/articles', () => {
             expect(articles).toBeSortedBy('created_at', {descending: true})
         })
     })
-})
\ No newline at end of file
+})
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('200: adds a comment with corresponding article_id', () => {
+        const testComment = {
+            username: 'butter_bridge',
+            body: 'Test comment',
+        };
+        return request(app)
+            .post('/api/articles/1/comments')
+            .send(testComment)
+            .expect(201)
+            .then((response) => {
+                expect(response.body.comment.author).toBe(testComment.username);
+                expect(response.body.comment.body).toBe(testComment.body);
+            });
+    });
+    test('404: returns appropriate error message when passed an invalid article ID', () => {
+        const testComment = {
+            username: 'butter_bridge',
+            body: 'Test comment',
+        };
+        return request(app)
+            .post('/api/articles/888/comments')
+            .send(testComment)
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toBe(`No article found for article id: 888`);
+            });
+    });
+    test('400: returns an error when a required field is missing in the comment', () => {
+        const incompleteComment = {
+            username: 'butter_bridge',
+        };
+        return request(app)
+            .post('/api/articles/1/comments')
+            .send(incompleteComment)
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe('Bad Request: Missing field');
+            });
+    });
+});
```

Comment:
We do need a 400 here as well for /api/articles/bananas or something like that. I know it's handled on your get, we're now on a post :)